foreman-installer/foreman-installer.spec

```diff
@@ -58,11 +58,14 @@ rake install \
   SYSCONFDIR=%{buildroot}%{_sysconfdir} \
   --trace
 
+%post
+foreman-installer --scenario foreman --migrations-only > /dev/null
```

Comment:
It should be run for each scenario. It is here for the configuration to get updated before users can touch it and is more of cosmetic thing. 
Installer ensures the migrations are run on its start anyway.